src/taipy/core/_version/_cli/_version_cli.py

```diff
@@ -0,0 +1,105 @@
+# Copyright 2023 Avaiga Private Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+# the License. You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+# an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+import sys
+
+from taipy.config._cli._argparser import _Argparser
+from taipy.logger._taipy_logger import _TaipyLogger
+
+from ...exceptions.exceptions import VersionIsNotProductionVersion
+from ...taipy import clean_all_entities_by_version
+from .._version_manager_factory import _VersionManagerFactory
+from ._bcolor import _Bcolors
+
+
+class _VersioningCLI:
```

Comment:
The name should be aligned with the module name.

Reply:
oops ...